tools/repology.json

```diff
@@ -4227,7 +4227,7 @@
     "name": "gn",
     "description": "GN is a meta-build system that generates build files for Ninja.",
     "homepage": "https://gn.googlesource.com/gn/",
-    "version": "dca877f",
+    "version": "b2afae1",
```

Comment:
Is there a real version that needs to go here?

Reply:
nope
![image](https://github.com/user-attachments/assets/93e85657-4b6f-4944-86f8-bbee9868d211)